.github/workflows/edge-test.yml

```
@@ -0,0 +1,101 @@
+  pull_request:
+    branches: [ '**' ]
+
+jobs:
+  edge-test:
+    name: Test Qdrant Edge
+
+    runs-on: ${{ matrix.os }}
+
+    # For merges into `dev` and `master`, run on all platforms.
+    # For PRs, run on ubuntu only to save CI time.
+    if: github.event_name != 'pull_request' || matrix.os == 'ubuntu-latest'
```

actionlint (1.7.11)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)